lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
```

Comment:
We should keep in mind that products is hardcoded around a bunch, so this will break pe stuff at least.